package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "root-eslint-plugin",
+  "name": "eslint-config-root-eslint-plugin",
```

Comment:
Should it maybe be `eslint-config-root` then? The `eslint-plugin` is probably redundant.